script/bootstrap

```diff
@@ -78,4 +78,11 @@ fi
 echo "  Application Bootstrap 2/2. Ansible (allthethings)..."
 ansible-playbook ${playbook} --inventory-file=${inventory} --connection=local
 
-echo "Bootstrap Complete"
\ No newline at end of file
+echo " Install MySQL 5.7 "
+sudo apt-get install mysql-server-5.7
```

Comment:
Also, we should be using RDS to run a mysql server, you likely don’t want to install the server inside the vagrant box itself.